machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -418,4 +422,74 @@ func main() {
 			log.ErrorExit(err)
 		}
 	}
+
+	if *flgBMCLogCollectorConfigMap {
+		// create bmc-log-collector configmap of all servers that include boot servers
+		addersses, err := updateServerAddressList(machines)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+		err = client.writeConfigMap(ctx, addersses)
```

Comment:
The function name `writeConfigMap()` is too abstract.
Please give it a name that clarifies the target resource.
For example, in the case of bmc-reverse-proxy, it is `client.updateBMCProxyConfigMap()`.

Also, all the functions that update the resources in this program are `client.updateHOGE()`.
But only this one is `client.writeHOGE()`. Please make the function names consistent.